### DISCUSSION TITLE
In my innertune there is no "playlist" filter

### DISCUSSION THREAD

**Discussion by max5354** [z-huang/music#411] [Q&A]
In my inner tune app there is no "playlists" filter. There is"feature playlists" and "community playlists" but there is no "playlists" filter option. I want to find my playlist.
![20221105_221743](https://user-images.githubusercontent.com/90886938/200131654-eb71005b-968e-4026-9d09-8b751c007a76.jpg)

**Comment by owenconnorz** [comment]
If your importing from yt music u have to make sure that the playlist u have is public and the  search your 1st and last name and it should come up

**Comment by z-huang** [comment]
Your playlist should appear in community playlists.
